### PR TITLE
Release notes for Foreman 3.4 RC1

### PR DESCRIPTION
**Pull request by upadhyeammit**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/doc-Release_Notes/master.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@ ifdef::katello[]
 
 include::topics/foreman.adoc[leveloffset=+1]
 include::topics/katello.adoc[leveloffset=+1]
+include::topics/foreman-3.4.0.adoc[leveloffset=+1]
```
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:13]: Should be outside of `ifdef::katello[]`.

File: guides/doc-Release_Notes/topics/foreman-3.4.0.adoc [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+* Missing smart-proxy-salt parameters in foreman-proxy scenario - https://projects.theforeman.org/issues/31502[#31502]
+
+=== Foreman modules
+
+* Insights client traffic through a Satellite 6.11 Capsule fails - https://projects.theforeman.org/issues/35278[#35278]
+* Autoindexing on /pub is broken because apache mod_autoindex is not loaded - https://projects.theforeman.org/issues/35227[#35227]
+
+=== External modules
+
+* cache-enabled setting for proxy content module isnt migrated to 'true' properlyt migrated to true' properlytrue properly properly - https://projects.theforeman.org/issues/35157[#35157]
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman-3.4.0.adoc:175]: I'm not sure what went wrong here. I do recall I had some troubles in the past with references in the script. Perhaps they're surfacing again.

Comment (upadhyeammit) [reply guides/doc-Release_Notes/topics/foreman-3.4.0.adoc]: yeah, there were few more, fixed those

File: guides/doc-Release_Notes/topics/foreman-3.4.0.adoc [hunk 3]
```diff
@@ -0,0 +1,210 @@
+= Foreman 3.4.0
+
+A full list of changes is available via https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1599[Redmine]
+
+== Foreman
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman-3.4.0.adoc:5]: I updated many categories on issues here. Would you mind regenerating?

**Review by ekohl — CHANGES_REQUESTED** [review]
Can you include the common parts in `foreman.adoc` from https://theforeman.org/manuals/3.4/index.html#Releasenotesfor3.4? So the headline features, upgrade warnings and deprecations?

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**